### PR TITLE
Adding messages showing that the tool is running

### PR DESCRIPTION
**Pull request by moozzyk**

### CODE REVIEW

**Comment by Tratcher** [comment]
Seems fine. Can you show a sample of the end-to-end publish output?

**Comment by moozzyk** [comment]
When running chained with `dotnet publish` it looks like this:
```
PS C:\Temp\WebApplication6\src\WebApplication6> dotnet publish -o C:\temp\WebApp6PublishOut\Debug\PublishOutput
Publishing WebApplication6 for DNXCore,Version=v5.0/win7-x64
Compiling WebApplication6 for DNXCore,Version=v5.0

Compilation succeeded.
    0 Warning(s)
    0 Error(s)

Time elapsed 00:00:02.9495408


Configuring project published to 'C:\temp\WebApp6PublishOut\Debug\PublishOutput' for use with IIS
Configuring project completed succesfully
Published to C:\temp\WebApp6PublishOut\Debug\PublishOutput
Publishing WebApplication6 for DNX,Version=v4.5.1/win7-x64
Compiling WebApplication6 for DNX,Version=v4.5.1

Compilation succeeded.
    0 Warning(s)
    0 Error(s)

Time elapsed 00:00:02.8064744


Configuring project published to 'C:\temp\WebApp6PublishOut\Debug\PublishOutput' for use with IIS
Configuring project completed succesfully
Published to C:\temp\WebApp6PublishOut\Debug\PublishOutput
Published 2/2 projects successfully
```

When running standalone it looks like this:
```
PS C:\Temp\WebApplication6\src\WebApplication6> dotnet publish-iis -p C:\Temp\WebApplication6\src\WebApplication6\bin\Debug\dnx451\win7-x64
Configuring project published to 'C:\Temp\WebApplication6\src\WebApplication6\bin\Debug\dnx451\win7-x64' for use with IIS
Configuring project completed succesfully
```

**Comment by Tratcher** [comment]
Maybe one more line of detail in the middle for the different situations?
```
Configuring project published to 'C:\temp\WebApp6PublishOut\Debug\PublishOutput' for use with IIS

No web.config found creating `C:\temp\WebApp6PublishOut\Debug\...\web.config`
or
Updating web.config at `C:\temp\WebApp6PublishOut\Debug\...\web.config`

Configuring project completed successfully
```

Also, succesfully -> successfully

File: src/dotnet-publish-iis/Program.cs [hunk 1]
```diff
@@ -32,7 +33,13 @@ public static int Main(string[] args)
                     return 2;
                 }
 
-               return new PublishIISCommand(publishFolder, projectPath.Value, webRootOption.Value()).Run();
+                Reporter.Output.WriteLine($"Configuring project published to '{publishFolder}' for use with IIS");
+
+                var exitCode = new PublishIISCommand(publishFolder, projectPath.Value, webRootOption.Value()).Run();
+
+                Reporter.Output.WriteLine("Configuring project completed succesfully");
```
Comment (muratg) [inline src/dotnet-publish-iis/Program.cs]: typo: succesSfully

**Comment by muratg** [comment]
:shipit:

File: src/dotnet-publish-iis/Program.cs (hunk 1 above)
Comment (moozzyk) [inline src/dotnet-publish-iis/Program.cs]: Yes. Will fix.

**Comment by moozzyk** [comment]
@Tratcher will add these messages

**Pull request closed without merging**